frontend/javascripts/viewer/view/action-bar/starting_job_modals.tsx

```diff
@@ -1054,9 +1067,10 @@ export function NucleiDetectionForm() {
 export function NeuronSegmentationForm() {
   const dataset = useWkSelector((state) => state.dataset);
   const { neuronInferralCostPerGVx } = features();
-  const hasSkeletonAnnotation = useWkSelector((state) => state.annotation.skeleton != null);
+  const skeletonAnnotation = useWkSelector((state) => state.annotation.skeleton);
   const dispatch = useDispatch();
   const [doSplitMergerEvaluation, setDoSplitMergerEvaluation] = React.useState(false);
+  const userAndTaskBoundingBoxes = useWkSelector((state) => getUserBoundingBoxesFromState(state)); // Includes task bounding boxes
```

Comment:
are you sure this contains task bboxes? the code doesn't look like that to me. I think, it would need to also call `getTaskBoundingBoxes`.

Reply:
I had the same thought. But during testing, I opened the annotation of a finished task and there was only the task bounding box in the bounding box tab. In the bounding box dropdown, this was selectable, and only the `userBoundingBoxes` are requested there. So I concluded that the task bounding box has to be converted to a user bbox somewhere? Additionally, the method `getTaskBoundingBoxes` is only called in the scene controller (not e.g. in the bounding box tab), so this felt consistent.

Reply:
the bbox tab only shows user bboxes, so it is expected that task bboxes won't be shown there. however, it's interesting that you saw a task bbox in the tab during testing. @fm3 do you know whether the backend converts task bboxes to user bboxes at some point? I don't think that this happens in the frontend.

Reply:
Oh okay, I see. Maybe I created a user bounding box within the task by accident then, because I just revalidated it and the task bbox didnt appear in the bounding box tab. I'll get the task bounding boxes then!